src/QuickCheck/PHPUnit/PropertyConstraint.php

```diff
@@ -18,18 +18,26 @@ class PropertyConstraint extends Constraint
      */
     private $seed;
 
+    private $showExceptionStacktrace = false;
+
     public function __construct(int $n, int $seed = null)
     {
         $this->numTests = $n;
         $this->seed = $seed;
     }
 
-    public static function check($n = 100, int $seed = null)
+    public static function check($n = 100, int $seed = null): self
     {
         return new self($n, $seed);
     }
 
-    public function evaluate($prop, string $description = '', bool $returnResult = false)
+    public function withExceptionStackrace(): self
```

Comment:
We are missing a `t` here in `Stackrace`.

Reply:
Good catch - thank you!